README.md

```diff
@@ -20,26 +20,52 @@ Or install it yourself as:
 
 ## Usage
 
-This API is provisional and is likely to change for 0.x versions.
+The API is provisional and is likely to change for versions < 1.0.0.
+
+To access the taxonomy, first request the content from the [publishing api](https://github.com/alphagov/publishing-api), then parse it to get a `LinkedEdition` object.
 
 ```ruby
 require 'plek'
 require 'gds_api/publishing_api'
 require 'govuk_taxonomy_helpers'
 
-content_store = GdsApi::ContentStore.new(Plek.new.find('content-store'))
-content_item = content_store.content_item('/education')
-taxonomy = GovukTaxonomyHelpers::Taxon.new(content_item)
+publishing_api = GdsApi::PublishingApi.new(Plek.new.find('publishing-api'))
```

Comment:
By using `Services.publishing_api` you can omit much of the plek/gds-api boilerplate.